app/javascript/components/Results/ResultGrid/ResultGrid.tsx

```diff
@@ -84,6 +85,13 @@ export const ResultGrid = ({ result, affiliate, query, position, vertical }: Res
           <div className='result-desc'>
             {getDescription(result.description)}
             <div className='result-url-text'>{truncateUrl(result.url, URL_LENGTH)}</div>
+            {/* Filter tags UI - to be updated with integration once backend starts sending the data */}
```

Comment:
What would need to be updated in here?

Reply:
Although the backend is not ready yet let's build everything as it where. Filter tags is an array, we should iterate through it here. Let's don't hardcode data like this rather add data into the server response.

Reply:
Once the backend tasks [SRCH-4983](https://cm-jira.usa.gov/browse/SRCH-4983) and [SRCH-4984](https://cm-jira.usa.gov/browse/SRCH-4984) will be ready, [SRCH-5067](https://cm-jira.usa.gov/browse/SRCH-5067) (facets integration) will be picked up to make the filter tags UI dynamic, currently it's a static UI.